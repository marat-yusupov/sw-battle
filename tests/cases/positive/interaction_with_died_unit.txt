CREATE_MAP 5 5 

SPAWN_WARRIOR 1 0 0 5 5

SPAWN_WARRIOR 2 1 1 5 0

SPAWN_WARRIOR 3 0 5 5 1

MARCH 1 5 0

MARCH 3 1 1