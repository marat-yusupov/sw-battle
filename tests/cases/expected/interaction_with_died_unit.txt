[1] MAP_CREATED width=5 height=5 
[1] UNIT_SPAWNED unitId=1 unitType=Warrior x=0 y=0 
[1] UNIT_SPAWNED unitId=2 unitType=Warrior x=1 y=1 
[1] UNIT_SPAWNED unitId=3 unitType=Warrior x=0 y=5 
[1] MARCH_STARTED unitId=1 x=0 y=0 targetX=5 targetY=0 
[1] MARCH_STARTED unitId=3 x=0 y=5 targetX=1 targetY=1 
[2] UNIT_ATTACKED attackerUnitId=1 targetUnitId=2 damage=5 targetHp=0 
[2] UNIT_DIED unitId=2 
[2] UNIT_ATTACKED attackerUnitId=2 targetUnitId=1 damage=0 targetHp=5 
[2] UNIT_MOVED unitId=3 x=1 y=4 
[3] UNIT_MOVED unitId=1 x=1 y=0 
[3] UNIT_MOVED unitId=3 x=1 y=3 
[4] UNIT_MOVED unitId=1 x=2 y=0 
[4] UNIT_MOVED unitId=3 x=1 y=2 
[5] UNIT_MOVED unitId=1 x=3 y=0 
[5] UNIT_MOVED unitId=3 x=1 y=1 
[5] MARCH_ENDED unitId=3 x=1 y=1 
[6] UNIT_MOVED unitId=1 x=4 y=0 
[7] UNIT_MOVED unitId=1 x=5 y=0 
[7] MARCH_ENDED unitId=1 x=5 y=0 
